store/src/java/com/zimbra/qa/unittest/TestDataSource.java

```diff
@@ -65,11 +76,15 @@
 import com.zimbra.soap.mail.type.NewFolderSpec;
 import com.zimbra.soap.type.DataSource.ConnectionType;
 
-public class TestDataSource extends TestCase {
+public class TestDataSource {
+    @Rule
+    public static TestName testInfo = new TestName();
 
-    private static final String USER_NAME = "user1";
+    private static String USER_NAME;
+    private static String USER_NAME_2;
     private static final String DS_NAME = "TestDataSource";
-    private static final String TEST_USER_NAME = "testdatasource";
+    private static final String TEST_USER_NAME = String.format("%s-%s-testuser1", TestDataSource.class.getSimpleName(), testInfo.getMethodName()).toLowerCase();
```

Comment:
Can't (meaningfully) use `testInfo.getMethodName()` in a static.
Make this non-final and initialize it in setUp()